lib/carelink/carelinkSimulator.js

```diff
@@ -1006,6 +1010,16 @@ exports.make = function(config){
      */
     getEvents: function(){
       return events;
+    },
+
+    /**
+     * Return a boolean reflecting whether or not this simulator instance has seen
```

Comment:
Should we explain why we care? If we are only using it because of the isolated issue that occurred should we explain that